docs/code_ref/net.rst

```diff
@@ -15,54 +15,71 @@ is an interface to multiple sources including all the sources implemented in
 
```

Comment:
Can you stick changes to this file in another PR so we can backport it?

Reply:
For all this to work it needs the changes in hek attrs as well, which is quite a lot (all be it could still be a bug fix).

Reply:
Done see #4834